### DISCUSSION TITLE
Lesson 13 with 0.8.23 and OZ 5  - using openzeppelin-foundry-upgrades

### DISCUSSION THREAD

**Discussion by billyjitsu** [Cyfrin/foundry-full-course-cu#1306] [Q&A]
Anybody having a working version of this tutorial with these updates?
I'm getting an error [FAIL. Reason: setup failed: custom error f92ee8a9:] 
When just trying to deploy BoxV1 during the initialize

**Comment by operation-c** [comment]
Paste your code. We need more context to help you.

**Comment by EngrPips** [comment]
Hey @billyjitsu, Have you resolved this issue already?
